src/nvim/window.c

```diff
@@ -1761,14 +1759,18 @@ bool one_window(void)
   return true;
 }
 
-/*
- * Close the possibly last window in a tab page.
- * Returns TRUE when the window was closed already.
- */
-static int close_last_window_tabpage(win_T *win, int free_buf, tabpage_T *prev_curtab)
+/// Close the possibly last window in a tab page.
+///
+/// @param win         The window to close
+/// @param free_buf    Whether to free the window's current buffer
+/// @param prev_curtab The tabpage to go to if this is the last window
```

Comment:
To me it seems we might be going to the alternative tabpage, and `prev_curtab` is the tabpage that might get closed if "win" is its last window.